content/en/docs/tasks/administer-cluster/manage-resources/quota-memory-cpu-namespace.md

```diff
@@ -141,6 +141,17 @@ Error from server (Forbidden): error when creating "examples/admin/resource/quot
 pods "quota-mem-cpu-demo-2" is forbidden: exceeded quota: mem-cpu-demo,
 requested: requests.memory=700Mi,used: requests.memory=600Mi, limited: requests.memory=1Gi
 ```
+{{< note >}}
+This is an exception for enforcing resource quota on ephemeral-storage.
```

Comment:
Is this an exception, or are we describing Kubernetes normal mechanism for resource requests and limits?

Reply:
@sftim  I have checked with other resources i.e CPU and Memory , we explicitly need to mention these values in pod spec, if we don't it throw as `Error creating: pods "ephpod-668668fc76-4rm6p" is forbidden: failed quota: ephtest: must specify limits.cpu` . But When i am trying with ephemeral-storage and don't provide request/limit in pod spec then pod is created but resource quota not enforced i.e i can use any number of ephemeral storage . So unless we provided request/limit in pod spec , resource quota is not enforced on ephemeral storage.

Reply:
This is -> There is ?

Reply:
> I have checked with other resources i.e CPU and Memory , we explicitly need to mention these values in pod spec, if we don't it throw as `Error creating: pods "ephpod-668668fc76-4rm6p" is forbidden: failed quota: ephtest: must specify limits.cpu` . But When i am trying with ephemeral-storage and don't provide request/limit in pod spec then pod is created but resource quota not enforced i.e i can use any number of ephemeral storage . So unless we provided request/limit in pod spec , resource quota is not enforced on ephemeral storage.

That sounds like there might be a LimitRange or other restriction. As far as I know a ResourceQuota _on its own_ doesn't force you to specify CPU or memory resources (I might be wrong and haven't checked).